spec/lib/sem/cli/orgs_spec.rb

```diff
@@ -34,13 +66,49 @@
   end
 
   describe "#members" do
+    let(:user_1) do
+      { :id => "ijovan", :permission => "write" }
+    end
+
+    let(:user_2) do
+      { :id => "shiroyasha", :permission => "admin" }
+    end
+
+    before do
+      allow(Sem::API::Orgs).to receive(:list_users).and_return([user_1, user_2])
+      allow(Sem::API::Orgs).to receive(:list_admins).and_return([user_2])
+      allow(Sem::API::Orgs).to receive(:list_owners).and_return([user_2])
+    end
+
+    it "calls the API" do
+      expect(Sem::API::Orgs).to receive(:list_users).with("renderedtext")
+
+      sem_run("orgs:members renderedtext")
+    end
+
+    context "admins option is true" do
+      it "calls the API" do
+        expect(Sem::API::Orgs).to receive(:list_admins).with("renderedtext")
+
+        sem_run("orgs:members renderedtext --admins true")
+      end
+    end
+
+    context "owners option is true" do
+      it "calls the API" do
+        expect(Sem::API::Orgs).to receive(:list_owners).with("renderedtext")
+
+        sem_run("orgs:members renderedtext --owners true")
+      end
+    end
+
     it "list members in an organization" do
       stdout, stderr = sem_run("orgs:members renderedtext")
 
       msg = [
-        "ID                                    NAME        PERMISSION  2FA",
-        "3bc7ed43-ac8a-487e-b488-c38bc757a034  ijovan      write       true",
-        "fe3624cf-0cea-4d87-9dde-cb9ddacfefc0  shiroyasha  admin       true"
+        "NAME",
+        "ijovan",
+        "shiroyasha"
```

Comment:
Forget it, I see. User can belong to many teams, and each team has a different permission. :/